src/operator/operator_common.h

```diff
@@ -314,6 +314,16 @@ inline bool dispatch_mode_assign(DispatchMode *y, const DispatchMode& x) {
   }
 #endif
 
+/* \brief message for storage fallback event */
+#define STORAGE_FALLBACK_MSG "You can set environment variable "               \
```

Comment:
Prefer `const std::string` over `#define` in c++.
https://stackoverflow.com/questions/5545570/const-string-vs-define